src/unix/linux_like/linux/mod.rs

```diff
@@ -322,6 +322,11 @@ s! {
     }
 
     pub struct input_event {
+        #[cfg(musl_time64_abi)]
+        pub input_event_sec: ::c_ulong,
+        #[cfg(musl_time64_abi)]
+        pub input_event_usec: ::c_ulong,
+        #[cfg(not(musl_time64_abi))]
         pub time: ::timeval,
```

Comment:
As mentioned above this should make use of `linux_time_bits64`. Specifically to match https://github.com/torvalds/linux/blob/bf9aa14fc523d2763fc9a10672a709224e8fcaf4/include/uapi/linux/input.h#L29 it could be:

```rust
#[cfg(any(target_pointer_width = "64", not(linux_time_bits64))]
pub time: ::timeval,

#[cfg(not(any(target_pointer_width = "64", not(linux_time_bits64)))]
pub input_event_sec: ::c_ulong,
#[cfg(all(not(any(target_pointer_width = "64", not(linux_time_bits64))), target_arch = "sparc64")]
pub input_event_usec: ::c_uint,
#[cfg(all(not(any(target_pointer_width = "64", not(linux_time_bits64))), target_arch = "sparc64")]
pub __pad: ::c_uint,
#[cfg(all(not(any(target_pointer_width = "64", not(linux_time_bits64))), not(target_arch = "sparc64"))]
pub input_event_usec: ::c_ulong,
```

unfortunately a bit messy but I think that's accurate